.github/workflows/ansible-lint-sap_general_preconfigure.yml

```
@@ -24,15 +24,19 @@ jobs:
 
     steps:
       - name: Check out the code
-        uses: actions/checkout@v2
+        uses: actions/checkout@main
 
       - name: Set up Python 3
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@main
```

```suggestion
        uses: actions/setup-python@v4
```
same argument as above